controller/common/common.go

```diff
@@ -190,3 +190,10 @@ func (m InformerMap) Get(apiVersion, resource string) *dynamicinformer.ResourceI
 func informerMapKey(apiVersion, resource string) string {
 	return fmt.Sprintf("%s.%s", resource, apiVersion)
 }
+
+func GetParent(informer *dynamicinformer.ResourceInformer, namespace, name string) (*unstructured.Unstructured, error) {
```

Comment:
What is the meaning of GetParent here?
Can we instead expose this method against dynamicinformer.ResourceInformer struct?

Reply:
adjusted name and added description